packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -927,14 +927,13 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				 * Your license key has expired.
 				 *
 				 * If you used the Free Trial, you either need to switch to
-				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html),
-				 * or acquire one of our [commercial plans](https://ckeditor.com/pricing/) if you wish to keep using the Premium Features.
+				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html), or
+				 * in case of commercial plan, change the trial key to production or development.
```

Comment:
Just a suggestion 

`trial key to production key or development key.` To make more explicit what we mean.

Reply:
In commit.